okio/jvm/src/main/java/okio/Buffer.kt

```diff
@@ -46,10 +46,9 @@ import javax.crypto.spec.SecretKeySpec
  */
 class Buffer : BufferedSource, BufferedSink, Cloneable, ByteChannel {
   @JvmField internal var head: Segment? = null
-  @JvmField internal var size: Long = 0L
 
-  /** Returns the number of bytes currently in this buffer.  */
-  fun size() = size
+  @get:JvmName("size")
+  var size: Long = 0L
```

Comment:
This makes it public?

Reply:
Added `private set` to suppress the public setter!